packages/react-dom/src/client/ReactDOMRoot.js

```diff
@@ -128,7 +129,18 @@ function createRootImpl(
       options.hydrationOptions != null &&
       options.hydrationOptions.mutableSources) ||
     null;
-  const root = createContainer(container, tag, hydrate, hydrationCallbacks);
+  const strictModeLevelOverride =
+    options != null && options.unstable_strictModeLevel != null
```

Comment:
Just wondering, can't we simply check if `options` is truthy here?

```suggestion
    options && options.unstable_strictModeLevel != null
```

Reply:
I could, but is there a strong reason to?

Truthy checks can be nice but they can also burn you sometimes so with code like this I tend to prefer a more explicit check.

Reply:
OK, thank you for explanation.

Reply:
No prob :)